tools/k8s/remote_connect.py

```diff
@@ -0,0 +1,366 @@
+#!/usr/bin/env python
```

Comment:
another thing that would be awesome is if we didn't need to configure it as much and it just "worked"

maybe could store the ssh key in google cloud so we could just download it without needing to set up the ssh key path.

maybe we could default the kubeconfig and other things to current context

i'm happy with this as if if you think this could be done as a follow up

Reply:
good idea. we chatted offline. I added defaults to all the config options but the ssh key and the proxy host.
thanks for the review